### PR TITLE
docs: update some docs with the right information

### PR DESCRIPTION
**Pull request by tariq1890**
Minor doc improvements

### CODE REVIEW

File: docs/clusterdefinition.md [hunk 1]
```diff
@@ -161,8 +161,9 @@ More usefully, let's add some custom configuration to the above addons:
               }
             ],
             "config": {
-              "maxNodes": "5",
-              "minNodes": "1"
+              "max-nodes": "5",
+              "min-nodes": "1",
+              "scan-interval": "10s"
```
Comment (mboersma) [inline docs/clusterdefinition.md]: There's another instance of `maxNodes` in cluster-autoscaler config at https://github.com/Azure/aks-engine/blob/master/pkg/engine/testdata/addons/kubernetes.json#L39-L54 that maybe should be fixed up similarly.

File: pkg/engine/testdata/addons/kubernetes.json [hunk 2]
```diff
@@ -129,8 +130,12 @@
       "etcdClientPrivateKey": "etcdClientPrivateKey",
       "etcdServerCertificate": "etcdServerCertificate",
       "etcdServerPrivateKey": "etcdServerPrivateKey",
-      "etcdPeerCertificates": ["etcdPeerCertificate0"],
-      "etcdPeerPrivateKeys": ["etcdPeerPrivateKey0"]
+      "etcdPeerCertificates": [
```
Comment (tariq1890) [inline pkg/engine/testdata/addons/kubernetes.json]: added formatting bonus

**Comment by CecileRobertMichon** [comment]
I've seen`docs` used in https://www.conventionalcommits.org/en/v1.0.0-beta.2/. @mboersma thoughts?

**Comment by mboersma** [comment]
:+1: for `docs:`. Let's use the standard "angular" types listed [here](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional#rules):

```
  'build',
  'ci',
  'chore',
  'docs',
  'feat',
  'fix',
  'perf',
  'refactor',
  'revert',
  'style',
  'test'
```

**Comment by tariq1890** [comment]
Fixed commit message and PR title @CecileRobertMichon @mboersma

**Pull request merged**